ui/src/common/ToggleFieldsMenu.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import IconButton from '@material-ui/core/IconButton'
+import Menu from '@material-ui/core/Menu'
+import MenuItem from '@material-ui/core/MenuItem'
+import MoreVertIcon from '@material-ui/icons/MoreVert'
+import Checkbox from '@material-ui/core/Checkbox'
+import { useDispatch, useSelector } from 'react-redux'
+import { useTranslate } from 'react-admin'
+import { setToggleableFields } from '../actions'
+
+const ITEM_HEIGHT = 70
+
+export default function ToggleFieldsMenu({ resource }) {
```

Comment:
Maybe set propTypes for this, marking `resource` as Required?

Reply:
Yeah. sure